version/version.go

```diff
@@ -1,15 +1,13 @@
 package version
 
-var (
-	TMCoreSemVer = TMVersionDefault
-)
+var TMCoreSemVer = TMVersionDefault
 
 const (
 	// TMVersionDefault is the used as the fallback version of Tendermint Core
 	// when not using git describe. It is formatted with semantic versioning.
-	TMVersionDefault = "0.34.20"
-	// ABCISemVer is the semantic version of the ABCI library
-	ABCISemVer = "0.18.0"
+	TMVersionDefault = "0.37.0"
```

Comment:
Since the version we're going to cut will be `v0.37.0-alpha.1`:

```suggestion
	TMVersionDefault = "0.37.0-alpha.1"
```

Reply:
Absolutely